script/app-coverage-report.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const path = require('path');
+const Table = require('cli-table');
+
+const printCoverage = coverageResults => {
+  // Create a new table with headers
+  const coverageTable = new Table({
+    head: ['Application', 'Lines', 'Functions', 'Statements', 'Branches'],
+  });
+
+  // Add each app coverage result to the table
+  Object.keys(coverageResults).forEach(key => {
+    if (key !== 'total') {
+      coverageTable.push({
+        [key]: [
+          `${coverageResults[key].lines.pct}%`,
+          `${coverageResults[key].functions.pct}%`,
+          `${coverageResults[key].statements.pct}%`,
+          `${coverageResults[key].branches.pct}%`,
+        ],
+      });
+    }
+  });
+
+  console.log(coverageTable.toString()); // eslint-disable-line
+};
+
+const generateCoverage = (rootDir, coverageSummary) =>
+  Object.keys(coverageSummary).reduce((acc, coverageResult) => {
+    const appName = coverageResult.replace(rootDir, '').split(path.sep)[0];
```

Comment:
This is good as a first pass, but we'll want to take a more nuanced approach before socializing this change.

Consider:
`src/applications/disability-benefits/` contains the following applications:
- 2346
- 526EZ
- 686
- 996
- all-claims
- disability-rating-calculator
- new-686

For the next pass, we should probably fetch all the `manifest.json` files and create a report for each individual application that way.